project/plugins.sbt

```diff
@@ -26,4 +26,5 @@ addSbtPlugin("net.bzzt" % "sbt-reproducible-builds" % "0.30")
 
 addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.10")
 addSbtPlugin("com.lightbend.sbt" % "sbt-publish-rsync" % "0.2")
+addSbtPlugin("com.github.pjfanning" % "sbt-source-dist" % "0.1.2")
```

Comment:
0.1.3 is ready - it generates the sha256/sha512 digest files as well as the zip/tgz